test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -3118,15 +3118,5 @@ bundle.js  2.84 KiB       0  [emitted]  main
 Entrypoint main = bundle.js
 [0] ./index.js 299 bytes {0} [built]
 [1] ./a.js 249 bytes {0} [built]
-[2] (webpack)/buildin/module.js 497 bytes {0} [built]
-
-WARNING in UglifyJs Plugin: Dropping unused function someUnRemoteUsedFunction1 [./a.js:3,0] in bundle.js
-
-WARNING in UglifyJs Plugin: Dropping unused function someUnRemoteUsedFunction2 [./a.js:4,0] in bundle.js
-
-WARNING in UglifyJs Plugin: Dropping unused function someUnRemoteUsedFunction3 [./a.js:5,0] in bundle.js
-
-WARNING in UglifyJs Plugin: Dropping unused function someUnRemoteUsedFunction4 [./a.js:6,0] in bundle.js
-
-WARNING in UglifyJs Plugin: Dropping unused function someUnRemoteUsedFunction5 [./a.js:7,0] in bundle.js"
+[2] (webpack)/buildin/module.js 497 bytes {0} [built]"
```

Comment:
Could you rename `test/statsCases/warnings-uglifyjs` too?

Reply:
I also wonder why this snapshot differs?

Reply:
> I also wonder why this snapshot differs?

It seems that the warnings message format is slightly different between the two plugins:
https://github.com/webpack-contrib/uglifyjs-webpack-plugin/blob/v1.3.0/src/index.js#L121
https://github.com/webpack-contrib/terser-webpack-plugin/blob/v1.1.0/src/index.js#L150

@evilebottnawi is this expected?

Reply:
@edmorley yep